datalake/src/main/java/com/sequenceiq/datalake/service/upgrade/SdxRuntimeUpgradeService.java

```diff
@@ -174,10 +174,10 @@ private FlowIdentifier triggerDatalakeUpgradeFlow(SdxUpgradeRequest request, Sdx
     }
 
     private void validateRollingUpgrade(SdxUpgradeRequest request, SdxCluster cluster) {
-        if (Boolean.TRUE.equals(request.getRollingUpgradeEnabled()) && !SdxClusterShape.MEDIUM_DUTY_HA.equals(cluster.getClusterShape())) {
+        if (Boolean.TRUE.equals(request.getRollingUpgradeEnabled()) && !cluster.getClusterShape().isHA()) {
             String message = String.format("The rolling upgrade is not allowed for %s cluster shape. "
                     + "If you want to use rolling upgrade you need to launch a Data Lake with %s cluster shape.",
-                    cluster.getClusterShape().name(), SdxClusterShape.MEDIUM_DUTY_HA.name());
+                    cluster.getClusterShape().name(), SdxClusterShape.SCALABLE.name());
```

Comment:
this message will be printed in case of MEDIUM_DUTY  too as I see, thus this should be fixed

Reply:
or will we recommend scalable instead of medium duty every time?

Reply:
I think for now we will keep this as medium duty, but in the future we can change this to scalable as that will be our ideal shape for everyone.
As for your question about upgrade/resize. Upgrade should function normally within this shape, and resize is a very simply change for this but is not part of this change. Upgrade has not been tested so we cannot confirm its functionality just yet.

Reply:
@kkalvagadda1 that does not apply to this case, we are recommending a specific cluster shape here.